test/BUILD.gn

```diff
@@ -223,7 +223,7 @@ test("brave_unit_tests") {
   if (enable_extensions) {
     sources += [
       "//brave/chromium_src/extensions/browser/sandboxed_unpacker_unittest.cc",
-      "//brave/chromium_src/extensions/common/permissions/permissions_data_unittest.cc",
+      "//brave/browser/extensions/api/brave_extensions_api_client_unittest.cc",
```

Comment:
looks like we're missing a dep for `//brave/browser/extensions` here and also `//extensions/browser/api/web_request` and `//url` for these https://github.com/brave/brave-core/pull/4057/files#diff-c88ad924973bbe2fe63d6025c3d19786R10
in general please always double check to make sure you have deps for any headers you add

Reply:
Added in last push.